WordPress/src/main/res/values/dimens.xml

```diff
@@ -677,6 +677,8 @@
     <dimen name="login_prologue_content_area">320dp</dimen>
     <dimen name="login_prologue_background_circle_size">140dp</dimen>
     <dimen name="login_prologue_background_circle_size_large">160dp</dimen>
+    <dimen name="login_prologue_revamped_buttons_padding">32dp</dimen>
+    <dimen name="login_prologue_revamped_buttons_padding_tablet">250dp</dimen>
```

Comment:
Not suggesting a change here, but curious why we have an extra layer of indirection here for WordPress (e.g. for the Jetpack value, we supply it directly in the tablet file). :thinking:

Reply:
Updated in https://github.com/wordpress-mobile/WordPress-Android/pull/17326/commits/e13662fc69439ce305c25197c1bfe29614f0023d